examples/llil_parser/src/LowLevel_IL_Parser.cpp

```diff
@@ -0,0 +1,433 @@
+/*
+LLIL Parser - Binary Ninja C++ API Sample
+ - Robert Yates - 22/JUN/17
+ */
+
+#include "LowLevel_IL_Parser.h"
+#include <iostream>
+#include <sstream>
+
+int main(int argc, char* argv[])
+{
+
+	try
+	{
+		ShowBanner();
+
+
+		if (argc != 2)
+		{
+			printf("Usage: %s <input file>\n", argv[0]);
+			exit(-1);
+		}
+
+		std::string inputName = argv[1];
+
+		SetBundledPluginDirectory(get_plugins_directory());
+		InitCorePlugins();
+		InitUserPlugins();
+
+		auto bd = BinaryData(new FileMetadata(), inputName.c_str());
+		BinaryView *bv;
+
+		for (auto type : BinaryViewType::GetViewTypes())
+		{
+			if (type->IsTypeValidForData(&bd) && type->GetName() != "Raw")
+			{
+				bv = type->Create(&bd);
+				break;
+			}
+		}
+
+		printf("[i] Starting analysis\n");
+		bv->UpdateAnalysis();
```

Comment:
thanks but i dont see this in the c++ class, or did you mean i should add this because you will be adding it soon? cheers /R.